templates/documents/general_view.html

```diff
@@ -109,16 +109,24 @@
 <table valign="top" width="100%">
     <tr>
         <td>
+            <div style="margin-bottom: 6px;padding-bottom: 6px;border-bottom:3px solid gray;">
+            <a class="css_button">{$file->get_url_web()}</a>
+            <label for='chkShow' class="text" style="margin-left:20px;"><b>{xl t='View contents'}</b>
```

Comment:
Not sure what would make sense from user perspective since it is bunch of stuff thrown together. So decided to carry forward the original term - just relocated the checkbox from middle of screen to top.  Soon we plan to replace top line by bootstrap's navbar with existing submit areas becoming modal popups.